<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>attrEditDialog</class>
 <widget class="QDialog" name="attrEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mapping</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../yoyirs.qrc">
    <normaloff>:/img/resources/logo.png</normaloff>:/img/resources/logo.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="SubtitleLabel" name="SubtitleLabel">
     <property name="text">
      <string>属性填写(Attribute Editing)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="shpSingleRenderScrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="ScrollAreaWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>473</width>
        <height>380</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2"/>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="PushButton" name="okPushButton">
       <property name="text">
        <string>确定(Space)</string>
       </property>
       <property name="shortcut">
        <string>Space</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="PushButton" name="cancelPushButton">
       <property name="text">
        <string>取消(Esc)</string>
       </property>
       <property name="shortcut">
        <string>Ctrl+S</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>SubtitleLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../yoyirs.qrc"/>
 </resources>
 <connections/>
</ui>
